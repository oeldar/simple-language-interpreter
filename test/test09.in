if 0 then
  print(10);
else
  print(20);
if 1 then
  print(10);
else
  print(20);
if 20 > 10 then print(20); else print(10);
if 20 < 10 then print(25); else print(20);
if 5 + 5 = 10 then print(30); else print(30);
if 5 + 5 <> 10 then print(35); else print(40);
