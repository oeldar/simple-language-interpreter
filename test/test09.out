20
10
20
20
30
40
